tests/functional/cylc-cat-log/15-remote_when_no_local_log.t

```diff
@@ -26,7 +26,7 @@ set_test_number 3
 create_test_global_config "" "
 [platforms]
    [[${CYLC_TEST_PLATFORM}]]
-       retrieve job logs = False"
+       retrieve job logs = True"
```

Comment:
Probs better to leave this as `False` and just remove the `rm` command from the workflow?